jsonrpc/endpoints_zkevm.go

```diff
@@ -204,6 +204,42 @@ func (z *ZKEVMEndpoints) GetBatchByNumber(batchNumber types.BatchNumber, fullTx
 	})
 }
 
+// GetBatchDataByNumbers returns the batch data for batches by numbers
+func (z *ZKEVMEndpoints) GetBatchDataByNumbers(filter types.BatchFilter) (interface{}, types.Error) {
```

Comment:
Nitpick, should we rename this to `GetBatchesByNumbers` (such that function name is `zkevm_getBatchesByNumbers`), and it gets analogue to the `GetBatchByNumber` (`zkevm_getBatchByNumber`). WDYT?

Reply:
it is actually returning only the data part of the batch, not the whole batch struct

Reply:
note @Stefan-Ethernal & @arnaubennassar I've rolled in a fix for ticket https://polygon.atlassian.net/browse/CDK-119 to this PR